generators/typescript-v2/ast/src/ast/__test__/Function.test.ts

```diff
@@ -0,0 +1,63 @@
+import { ts } from "../..";
+
+describe("Invocation", () => {
+    describe("invokeFunction", () => {
+        it("Should generate an invokeFunction", () => {
+            const actual = ts.invokeFunction({
+                func: ts.reference({ name: "WithBaseURL", importPath: "fern" }),
+                arguments_: [ts.TypeLiteral.string("https://api.example.com")]
+            });
+            expect(actual.toStringFormatted()).toMatchSnapshot();
+        });
+    });
+
+    describe("function", () => {
+        it("Should generate a function", () => {
+            const actual = ts.func({
+                name: "WithBaseURL",
```

Comment:
nit: For all these tests, let's use `withBaseURL` so that it's more idiomatic TypeScript. Exported functions are often `camelCase`'d in TypeScript.